src/lib/work_selector/work_lib.ml

```diff
@@ -6,24 +6,14 @@ module Make (Inputs : Intf.Inputs_intf) = struct
   module Inputs = Inputs
   module Work_spec = Snark_work_lib.Work.Single.Spec
 
-  module Job_status = struct
-    type t = Assigned of Time.t
-
-    let is_old (Assigned at_time) ~now ~reassignment_wait =
-      let max_age = Time.Span.of_ms (Float.of_int reassignment_wait) in
-      let delta = Time.diff now at_time in
-      Time.Span.( > ) delta max_age
-  end
-
   module State = struct
-    module Seen_key = struct
-      module T = struct
-        type t = Transaction_snark.Statement.t One_or_two.t
-        [@@deriving compare, sexp, to_yojson, hash]
-      end
+    (* TODO: maybe factor out job numbering mechanism from partitioner to here,
```

Comment:
Not sure if there is a straightforward way to use numbering here.

Reply:
Indeed.